uportal-war/src/main/data/default_entities/permission_owner/UP_USERS.permission-owner.xml

```diff
@@ -45,7 +45,7 @@
         <name>Edit user attribute</name>
         <fname>EDIT_USER_ATTRIBUTE</fname>
         <desc>Edit a specific user attribute.  This permission applies to that attribute for all editable users</desc>
-        <targetProvider>peopleAndGroupsTargetProvider</targetProvider>
+        <targetProvider>userAttributesTargetProvider</targetProvider>
```

Comment:
FYI I believe from the comment that the target is an attribute name, not a person who can edit all attributes, so that's why I changed this.  I.e. it is similar to VIEW_USER_ATTRIBUTE.